tests/Bridge/Symfony/Bundle/Test/ApiTestCaseTest.php

```diff
@@ -24,6 +24,27 @@
 
 class ApiTestCaseTest extends ApiTestCase
 {
+    public function testCreateClientBootsKernelIfNotPresent(): void
+    {
+        $this->assertNull(static::$kernel, "Kernel not booted");
```

Comment:
Not sure what to do with this check failure.
I want to make sure, that the preconditions for my tests are fulfilled. If somebody later changes the code that a static::$kernel === null is no longer the default-value, my change would break. So I included this in the test to make it bullet-proof ...

Reply:
Don't worry about this one, it's unrelated with your PR and is being fixed in #3702.